### ISSUE TITLE
App going down when accessing Summary admin page

### ISSUE THREAD

**Issue by schlos** [codeforcroatia/Reklamacije.net#21]
### Steps to reproduce

1. open Summary page in Admin

### Expected behaviour
_Tell us what should happen:_

Should open Summary page.

### Actual behaviour
_Tell us what happens instead:_

App going down when accessing Summary admin page

### Application version

> Alaveteli version: 0.28.0.9 
> Alaveteli branch: master 
> Alaveteli commit: 588b4014057128cb4cc74098752208d996692745 
> RUBY_VERSION 1.9.3 
> Rails::VERSION::STRING 4.0.13 
> Xapian::version_string 1.2.21

### Client configuration

**Full URL:**

**Operating system**: (optional)

**Browser client:** (optional)

**Browser version:** (optional)

#### Browser log (optional)
```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log 
c) ...
```

**Comment by schlos** [comment]
Known fix for this:
Louise made the hotfix for this issue (summary restart issue) - it’s release 0.28.0.14, with the same tag (https://github.com/mysociety/alaveteli/releases/tag/0.28.0.14)

**Comment by schlos** [comment]
Live application is patched to 0.28.0.14 version. Release done on 02-JAN-2018 at 20:30h.
